app/views/root/_publication.html.erb

```diff
@@ -12,7 +12,7 @@
 
     <% if publication.published? %>
       <%= link_to "#{Plek.new.website_root}/#{publication.slug}", class: 'link-muted' %>
-    <% elsif publication.safe_to_preview? %>
+    <% elsif !publication.archived? %>
       <%= link_to "/#{publication.slug}", preview_edition_path(publication), class: 'link-muted' %>
     <% else %>
       <span class="text-muted"><%= publication.slug %></span>
```

Comment:
Now, if the publication is archived, this will just display (as per the screenshot) the plain slug (without linking to anything). Should the slug disappear entirely in this case?